pkg/rehearse/jobs.go

```diff
@@ -183,6 +204,11 @@ func ConfigureRehearsalJobs(toBeRehearsed config.Presubmits, ciopConfigs config.
 				continue
 			}
 
+			exists, index, templateName := hasChangedTemplateVolume(rehearsal.Spec.Containers[0].VolumeMounts, rehearsal.Spec.Volumes, templates)
+			if exists {
+				rehearsal.Spec.Volumes[index].VolumeSource.ConfigMap.Name = config.GetTempCMName(prNumber, templateName)
```

Comment:
How do we test this?

Reply:
`TestConfigureRehearsalJobs` should be updated. This PR is already getting bigger and bigger. I would like to do this in a follow-up